.travis.yml

```diff
@@ -21,11 +21,15 @@ before_script:
       pip install 'travis-cargo<0.2' --user &&
       export PATH=$HOME/.local/bin:$PATH
 
+# script:
+#   - |
+#       travis-cargo build &&
+#       travis-cargo test
 script:
   - |
-      travis-cargo build &&
-      travis-cargo test
-after_success:
+      travis-cargo build
+
+      after_success:
   - travis-cargo coveralls --no-sudo --verify
   - ./kcov/build/src/kcov --verify --coveralls-id=$TRAVIS_JOB_ID --exclude-pattern=/.cargo target/kcov target/debug/lib-*
```

Comment:
I guess those two lines (coveralls and kcov) should be commented as well in order for the tests to pass.

Reply:
I try this out. Sorry for the noise.

Reply:
Don't worry about that: thanks for your work.
I believe you should remove `&&` after `travis-cargo build` and don't forget to comment the `after_success` section as it executes the tests as well, which won't work.